app/page/itinerary.cjsx

```diff
@@ -19,78 +14,57 @@ class ItineraryPage extends React.Component
-        from = geo_string
-      else
-        from = locationToOTP(origin)
-
-      if destination.useCurrentPosition
-        to = geo_string
-      else
-        to = locationToOTP(destination)
-      setTimeout(() =>
-        @context.router.replace getRoutePath(from, to) + "/" + index
-        itineraryTabState = @refs["itineraryTab" + index].getState()
-        @focusMap(itineraryTabState.lat, itineraryTabState.lon)
-      , 100)
-
-  getSlides: (itineraries) =>
-    slides = []
-    for itinerary, i in itineraries
-      slides.push <div className={"itinerary-slide-container"} key={i}>
-                    <ItineraryTab
-                    ref={"itineraryTab" + i}
-                    focus={@focusMap}
-                    itinerary={itinerary}
-                    index={i}/>
-                  </div>
-    slides
-
-  getTabs: (itineraries, selectedIndex) =>
-    tabs = []
-    for itinerary, i in itineraries
-      color = if i == selectedIndex then "#007ac9" else "#ddd"
-      tabs.push <Tab
-                  selected={if i == selectedIndex then true else false}
-                  key={i}
-                  label="•"
-                  value={i}
-                  className={if i == selectedIndex then "itinerary-tab-root--selected" else "itinerary-tab-root"}
-                  style={{height: "18px", color: color, fontSize: "34px", padding: "0px"}} />
-    tabs
-
   render: =>
-    plan = @context.getStore('ItinerarySearchStore').getData().plan
-    unless plan
-      return <DefaultNavigation className="fullscreen"/>
-    itineraries = plan.itineraries
-
-    leafletObj = <ItineraryLine key={"line" + @props.params.hash} legs={itineraries[parseInt(@props.params.hash)].legs} showFromToMarkers={true} showTransferLabels={true}/>
+    from = otpToLocation(@props.params.from)
+    to = otpToLocation(@props.params.to)
+
+    store = @context.getStore('ItinerarySearchStore')
+    modes = store.getMode()
+    walkReluctance = store.getWalkReluctance()
+    walkBoardCost = store.getWalkBoardCost()
+    minTransferTime = store.getMinTransferTime()
+    walkSpeed = store.getWalkSpeed()
+    wheelchair = store.isWheelchair()
+    if store.getMode().indexOf('BICYCLE') == -1
+      maxWalkDistance = config.maxWalkDistance
+    else
+      maxWalkDistance = config.maxBikingDistance
+
+    arriveBy = @context.getStore('TimeStore').getArriveBy()
+    selectedTime = @context.getStore('TimeStore').getSelectedTime()
+
+    preferredAgencies = config.preferredAgency or ""
+    disableRemainingWeightHeuristic = store.getCitybikeState()
+
+    plan = <Relay.RootContainer
+      Component={ItineraryPlanContainer}
+      route={new queries.SummaryPlanContainerRoute(
+        fromPlace: @props.params.from
+        toPlace: @props.params.to
+        numItineraries: 3
+        modes: modes
+        date: selectedTime.format("YYYY-MM-DD")
+        time: selectedTime.format("HH:mm:ss")
+        walkReluctance: walkReluctance + 0.000099
```

Comment:
Yes, the + 0.000099 is a big in graphql-java and will be removed once we can use the patched version. Patch has been merged already.